lab_3_keywords_textrank/main.py

```diff
@@ -81,7 +90,7 @@ def preprocess_text(self, text: str) -> tuple[str, ...]:
             tuple[str, ...]
                 clean lowercase tokens with no stop-words
         """
-        pass
+        return self._remove_stop_words(self._clean_and_tokenize(text))
```

Comment:
please, save the result of `self._clean_and_tokenize(text)` to a variable

Reply:
done